Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -251,6 +251,33 @@ extension AlertManager {
     }
 }
 
+// MARK: PersistedAlertStore
+extension AlertManager: PersistedAlertStore {
+    public func lookupOutstandingAlerts(managerIdentifier: String, completion: @escaping (Result<[PersistedAlert], Error>) -> Void) {
```

Comment:
[nit] like above where `Unacknowledged` and `Unretracted` are used, consider replacing `Outstanding` with terms already used for alerts (here maybe `Unretracted`)